packages/vite/src/vite.ts

```diff
@@ -25,6 +26,22 @@ export interface ViteBuildContext {
 }
 
 export async function bundle (nuxt: Nuxt) {
+  // https://github.com/vitejs/vite/blob/8fe69524d25d45290179175ba9b9956cbce87a91/packages/vite/src/node/constants.ts#L38
+  [
+    'vite.config.js',
+    'vite.config.mjs',
+    'vite.config.ts',
+    'vite.config.cjs',
+    'vite.config.mts',
+    'vite.config.cts'
+  ].some((fileName) => {
+    if (existsSync(resolve(nuxt.options.rootDir, fileName))) {
+      consola.warn(`\`${fileName}\` is not supported. Use \`options.vite\` instead. You can read more in \`https://nuxt.com/docs/api/configuration/nuxt-config#vite\`.`)
+      return true
+    }
+    return false
+  })
```

Comment:
> Here, we could use Vite's [loadConfigFromFile()](https://vitejs.dev/guide/api-javascript.html#loadconfigfromfile) instead

I mean, it will return `null` if there is no config file found.

Reply:
I think your approach is more minimal, and the warning is a 'nice-to-have' extra so it is fine if we don't detect an edge case.

You could consider using `resolvePath` from `mlly` or Nuxt Kit to resolve the config file which might simplify the implementation here.